### PR TITLE
Added CVE-2016-0808

### PR DESCRIPTION
**Pull request by kayji**
Hi,

This pull request contains the check for CVE-2016-0808

cve20160808.c in jni
CVE_2016_0808.java in framework/graphics
vuln_map.json
VulnerabilityOrganizer.java

### CODE REVIEW

File: android-vts.iml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Fuzion24) [inline android-vts.iml]: Can you please remove this file

File: app/app.iml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<module external.linked.project.id=":app" external.linked.project.path="$MODULE_DIR$" external.root.project.path="$MODULE_DIR$/.." external.system.id="GRADLE" external.system.module.group="androidVTS" external.system.module.version="unspecified" type="JAVA_MODULE" version="4">
+<module external.linked.project.id=":app" external.linked.project.path="$MODULE_DIR$" external.root.project.path="$MODULE_DIR$/.." external.system.id="GRADLE" external.system.module.group="android-vts" external.system.module.version="unspecified" type="JAVA_MODULE" version="4">
```
Comment (Fuzion24) [inline app/app.iml]: Also, changes to this file do not need to be checked in.

File: app/src/main/jni/cve20160808.c [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+	}
+
+	fseek(file, 0, SEEK_END);
+	size = ftell(file);
+	rewind(file);
+
+	buffer = (uint8_t *)malloc(sizeof(char)*size);
+
+	fread(buffer, 1, size, file);
+	uint8_t needle[4] = {0x53, 0x55, 0x55, 0x15};
```
Comment (Fuzion24) [inline app/src/main/jni/cve20160808.c]: What does this needle represent? some information about this, what the bug was, and how the patch works would be great.

**Comment by Fuzion24** [comment]
Thank you for your contribution.  I've made some inline comments.  Also, you may want to consider using binmatcher: https://github.com/nowsecure/android-vts/blob/master/app/src/main/java/fuzion24/device/vulnerability/vulnerabilities/framework/media/CVE_2015_6616.java#L61

This avoids having to use native code here, which i've been trying to do whenever reasonably possible

**Comment by kayji** [comment]
Thanks for your feedback!

I added some explanation about the details of this CVE in cve20160808.c.

I will try to avoid using native code next time.

**Comment by Fuzion24** [comment]
@kayji would you mind rebasing this on the current master?  Thanks.

**Comment by kayji** [comment]
I'll start working on that soon.

**Comment by Fuzion24** [comment]
awesome, thank you.

**Comment by m00head** [comment]
Not sure what happened to this. I have done a rebase on the current master branch here:

**Pull request closed without merging**